packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/Buttons/index.tsx

```diff
@@ -65,8 +64,9 @@ const Bottom = () => {
                   .dividedBy(divisor)
                   .decimalPlaces(network.decimals)
                   .toString();
-        setValue('sendCryptoInput', amount);
+        setValue(cryptoInput, amount);
         updateFiatValue(amount);
+        if (composeRequest) composeRequest();
```

Comment:
composeRequest undefined case, discuss

Reply:
Fixed, it is never undefined (also in useFees).